### ISSUE TITLE
Init should default to current project root

### ISSUE THREAD

**Issue by 50Wliu** [atom/github#1237]
### Prerequisites

* Have you tried to reproduce the problem in [Safe Mode][safe-mode]?
* Have you followed all applicable steps in the [debugging guide][debugging-guide]?
* Have you checked the FAQs on the [message board][message-board] for common solutions?
* Have you checked that your issue [isn't already filed][already-filed]?

[safe-mode]: http://flight-manual.atom.io/hacking-atom/sections/debugging/#using-safe-mode
[debugging-guide]: http://flight-manual.atom.io/hacking-atom/sections/debugging/
[message-board]: https://discuss.atom.io/c/faq
[already-filed]: https://github.com/atom/github/issues

### Description

Today I created a new folder and opened Atom in it, intending to start working on a new Git repository.  So I clicked on the 0 files icon and then tried to init a repository.  Instead of populating the location field with the project I had opened, it defaulted to `%USERPROFILE%\github`.

### Steps to Reproduce

1. Open a folder in Atom that is not a Git repository
2. Try to init through the github package

**Expected behavior:** Field should be pre-populated with current project path

**Actual behavior:** Populated with `C:\Users\<username>\github`

**Reproduces how often:** 100%

### Versions

```
Atom    : 1.23.0-dev-042c22f43
Electron: 1.6.15
Chrome  : 56.0.2924.87
Node    : 7.4.0
```

**Comment by simurai** [comment]
Ran into this too. Here a gif:

![init](https://user-images.githubusercontent.com/378023/34548951-64056e3c-f148-11e7-80a2-d6833688701e.gif)

Having to enter the project name seems unexpected and in my case I just clicked on "Init" which then created a Git repo with all my projects in the parent `github` directory.

Would it be possible to skip the second step? So if you click on the "Create repository" button, it would initialize in the current directory without having to type the path.
